history-of-quakers/modernized/25-chapter-23.adoc

```diff
@@ -863,6 +926,9 @@ that a bill was brought into parliament, for restraining the licentiousness of t
 Those called Quakers, perceiving that this might be pernicious,
 drew up the following remarks, which they delivered to the members of parliament:
 
+[.embedded-content-document.address]
+--
+
 Some considerations humbly offered by the people called Quakers,
```

Comment:
Again, this has a letter heading or heading feel, but I wasn't sure if it should be!